### PR TITLE
Turned off current survey and generalised surveygizmo template

### PR DESCRIPTION
**Pull request by ajparsons**
The current wtt proxy use survey is turned off.

The survey title and url are moved up to firsttime.php so all config is in the same file.

Old survey details left in place because the parameters are useful for next time

### CODE REVIEW

**Comment by ajparsons** [comment]
Added a replacement survey for people who wrote to local councillors

File: web/firsttime.php [hunk 1]
```diff
@@ -37,11 +37,25 @@
...
-// Proxy Information Survey
-
-$rand = rand(0, 1); // high rate when want lots of data
-if ($rand == 0) {
-    template_draw("surveygizmo-proxy", $values);
+// SurveyGizmo survey hook
+// Current url for local councillor name recognition survey
+// https://www.surveygizmo.com/s3/5057760/WTT-Name-Recognition-Local-Council
+
+local_council_codes = ['DIW','CED','LBW','COP','LGE','MTW','UTE','UTW'];
```
Comment (dracos) [inline web/firsttime.php]: This is missing a $ at the start.

File: web/firsttime.php [hunk 2]
```diff
@@ -37,11 +37,25 @@
...
+$wrote_to_councillor = in_array($values["recipient_type"], local_council_codes);
```
Comment (dracos) [inline web/firsttime.php]: and then here too, same var. Unless I've lost track of how PHP works :)

File: web/firsttime.php [hunk 3]
```diff
@@ -37,11 +37,25 @@
...
+
+$send_to_survey_gizmo = $wrote_to_councillor; 
+// use rand(0, 1); for a 50% referral rate 
+
+if ($send_to_survey_gizmo == 1) {
```
Comment (dracos) [inline web/firsttime.php]: As it's true/false, better to just have `if ($send_to_survey_gizmo) {`

File: templates/website/surveygizmo-survey.html [hunk 4]
```diff
@@ -22,4 +21,4 @@
     </div>
 </div>
 
-<?php template_draw('footer', $values);
+<?php template_draw('footer', $values);?>
```
Comment (dracos) [inline templates/website/surveygizmo-survey.html]: We don't have final `?>` in files, to prevent any ending-whitespace issues.

**Comment by ajparsons** [comment]
@dracos : Should have resolved the above.

**Pull request merged**